contribs/gmf/src/directives/mobilemeasurepoint.js

```diff
@@ -293,11 +294,11 @@ gmf.MobileMeasurePointController.prototype.getMeasure_ = function() {
         childEl.innerHTML = [this.translate(key), ': ', value, ' ', unit].join('');
         ctn.appendChild(childEl);
       }
-    }, this);
+    }
 
-    const previousCtn = goog.dom.getElementByClass(className, el);
-    if (previousCtn) {
-      previousCtn.remove();
+    const previousCtn = el.getElementsByClassName(className);
+    if (previousCtn.length > 0) {
```

Comment:
The test must be `if (previousCtn.length > 0 && previousCtn[0])`.
I am also fine with the shorter `if (previousCtn.length && previousCtn[0])`.